src/components/RadioButtonGroup/RadioButtonGroup.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { radioButtonGroupDriverFactory } from './RadioButtonGroup.driver';
+import { RadioButtonGroup } from './';
+import { radioButtonGroupTestkitFactory } from '../../testkit';
+import { radioButtonGroupTestkitFactory as enzymeRadioButtonGroupTestkitFactory } from '../../testkit/enzyme';
+import { RadioButton } from '../RadioButton';
+
+const noop = () => {};
+const TestComp: React.FunctionComponent<any> = props => (
+  <RadioButtonGroup {...props}>
+    <RadioButton value="group1" onChange={noop} label="Checkbox 1️⃣" />
+    <RadioButton value="group1" onChange={noop} label="Checkbox 2️⃣" />
+    <RadioButton value="group1" onChange={noop} label="Checkbox 3️⃣" />
+  </RadioButtonGroup>
+);
+
+describe('RadioButtonGroup', () => {
+  const createDriver = createUniDriverFactory(radioButtonGroupDriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(<RadioButtonGroup name={'name'} />);
+    expect(await driver.exists()).toBe(true);
+  });
+  it('should render with children', async () => {
+    const driver = createDriver(<TestComp />);
+
+    expect(await driver.exists()).toBe(true);
+    expect(await driver.isRadioButtonsExists()).toBe(true);
+  });
```

Comment:
I think it would be better to have a `getChildren` method in the driver
And then test it like this:
```suggestion
  it('should render with children', async () => {
    const driver = createDriver(<TestComp />);

    expect(await driver.exists()).toBe(true);
    expect((await driver.getChildren()).length).toBe(3);
    // you can add other expectations about the children here
  });
```

Reply:
ping for this ☝️ 
The name can be different though, maybe `getItems` or `getRadios`
In addition, we can also add an `isEmpty` method instead of `isRadioButtonsExists`.
So in fact:
```ts
isEmpty:  async () =>  {
  const items = await getItems();
  return !items.length;
}
```